qutebrowser/browser/hints.py

```diff
@@ -986,3 +1003,112 @@ def on_mode_left(self, mode):
             # hinting.
             return
         self._cleanup()
+
+
+class WordHinter(object):
```

Comment:
The `(object)` isn't needed as it's python3 only code, and for consistency (as it's not there everywhere else), change this to `class WordHinter:` please.